.github/workflows/ci.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
         include:
           - image: swift:5.5-focal
             env:
-              MAX_ALLOCS_ALLOWED_bidi_1k_rpcs_10_requests: 504000
+              MAX_ALLOCS_ALLOWED_bidi_1k_rpcs_10_requests: 504001
```

Comment:
In your PR description, you wrote that a recent PR in NIO reduced allocations, but this is increasing the limits. Or am I missing something?

Reply:
Yup, this was a no-op change to get the CI to run so we can extract the new limits (hence this is a draft).